habitat-baselines/habitat_baselines/config/habitat_baselines/rl/policy/hierarchical_policy/defined_skills/oracle_skills_drone_multi_agent.yaml

```diff
@@ -3,7 +3,9 @@
 
 wait:
   skill_name: "WaitSkillPolicy"
-  max_skill_steps: -1
+  action_name: "wait"
+  obs_skill_inputs: []
+  max_skill_steps: 50
```

Comment:
Well, things are different now, since new `WaitAction` expanded the action spaces to adapt to the `any_action()` check in [here](https://github.com/SgtVincent/habitat-lab/blob/39c56837074c9883a04d6c928405361f3b8b6810/habitat-baselines/habitat_baselines/rl/multi_agent/multi_llm_policy.py#L75). I'll try to fix it anyway.